pkg/oc/admin/policy/remove_from_project.go

```diff
@@ -176,7 +176,11 @@ func (o *RemoveFromProjectOptions) Run() error {
 		}
 
 		if !o.DryRun {
-			_, err = o.Client.RoleBindings(o.BindingNamespace).Update(&currBinding)
+			if len(currBinding.Subjects) > 0 {
+				_, err = o.Client.RoleBindings(o.BindingNamespace).Update(&currBinding)
+			} else {
+				err = o.Client.RoleBindings(o.BindingNamespace).Delete(currBinding.Name, &metav1.DeleteOptions{})
```

Comment:
These delete options should include the UID as a precondition.

Reply:
Hmm it is actually UID + RV that we want here.  @liggitt do we have a way to specify that via delete options?

Reply:
Preconditions doesn't support resource version today... no particular reason it couldn't

Reply:
I do not see why we want either UID or RV, most of these commands are not "safe" that way and they do not look they are meant to be or can be.

Reply:
Pinning uid and rv as preconditions would ensure you only deleted the object in its current state (another modification that added another subject would change the rv and make your delete fail on a conflict error, notifying you you were deleting data added by some other process)

Reply:
> no other command in oc adm policy does that fore removal

no other command injects data into found resources either

> so I do not think it makes sense to try to shoehorn that in this PR

agree it isn't required for this PR